src/main/scala/com/fulcrumgenomics/umi/UmiConsensusCaller.scala

```diff
@@ -253,6 +253,7 @@ trait UmiConsensusCaller[C <: SimpleRead] {
     * NOTE: filtered out reads are sent to the [[rejectRecords()]] method and do not need further handling
     */
   protected[umi] def filterToMostCommonAlignment(recs: Seq[SAMRecord]): Seq[SAMRecord] = {
+    require(recs.map(_.getReadNegativeStrandFlag).distinct.size <= 1, "Not all records were on the same strand.")
```

Comment:
In case we change this method somehow and the constraint goes away or is modified.  I want it _explicit_ in the duplex caller.